parachains/runtimes/assets/asset-hub-kusama/src/xcm_config.rs

```diff
@@ -567,13 +587,21 @@ pub mod bridging {
 						// allow receive DOT
 						.add_equals(DotLocation::get())
 				)
+			),
+			(
+				EthereumLocation::get(),
+				AssetFilter::ByMultiLocation(
+					MultiLocationFilter::default()
+						.add_starts_with(EthereumLocation::get())
+				) // TODO: Change to registry contract address. For now accept any ethereum address.
```

Comment:
Is it safe for us to trust any multilocation that is from Ethereum as a reserve location? I think so because only contracts that have SUBMIT_ROLE permission will be able to enqueue messages. However, another contract with SUBMIT_ROLE privilege could send through a Mint message and this predicate would not guard against that.